changelog.R

```diff
@@ -1,8 +1,18 @@
 output$changelog <- renderTable({
   tribble(
     ~Date, ~Change,
+    "04-09-2024", "Moved all variables from global environment to a session 
```

Comment:
Add entry: 05-02-2024, Added demo mode, where users can try out Eva even if they don't have access to an FY 2024 compliant HUD CSV Export. (or something similar)